lang/ar.yml

```diff
@@ -167,8 +167,6 @@ ar:
     many_many_BackLinkTracking: 'متابعة التعقيبات'
     many_many_ImageTracking: 'متابعة الصور'
     many_many_LinkTracking: 'متابعة الروابط'
-  SilverStripe\CMS\Model\SiteTreeFileExtension:
```

Comment:
Can you please restore ALL the translation updates - we cannot delete keys from the yml files.  If we do, when we push the changes to transifex the translations will be deleted from transifex, so if the translations are pulled for older versions of silverstripe, then the translations will be deleted from the yml files of the older version of silverstripe

Reply:
Done